manifests/python.yml

```diff
@@ -890,7 +890,11 @@ tests/:
       Test_Decisionless_Extraction: v2.8.0
     test_sampling_span_tags.py:
       Test_Sampling_Span_Tags: v2.8.0
-    test_span_events.py: missing_feature
+    test_span_events.py:
+      Test_SpanEvents_WithAgentSupport:
+        "*": irrelevant
```

Comment:
Can we document why these tests are not required (ex: this feature is enabled via an envar, it is independent of the agent version)